requirements.txt

```diff
@@ -2,5 +2,6 @@
 asn1crypto
 codicefiscale
 elementpath
+git+https://github.com/OCA/openupgradelib.git@master
```

Comment:
@SimoRubi You can try `openupgradelib @ git+https://github.com/OCA/openupgradelib.git@master`

Reply:
Thanks for having a look!
I tried your suggestion but pre-commit kindly suggests me to edit:
```
-openupgradelib @ "git+https://github.com/OCA/openupgradelib.git@master"
+git+https://github.com/OCA/openupgradelib.git@master
```
that in the end Travis doesn't like:
```
    pkg_resources.extern.packaging.requirements.InvalidRequirement: Parse error at "'+https:/'": Expected stringEnd
```
(from https://travis-ci.com/github/OCA/l10n-italy/jobs/521127703#L567)